distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc

```diff
@@ -1,94 +1,107 @@
+:context: distributed-tracing-rn-3-1
 
 toc::[]
 
 include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
 
-[id="component-versions_distributed-tracing-rn-3-0"]
-== Component versions in the {DTProductName} 3.0
+[id="distributed-tracing-rn_3-1_component-versions"]
+== Component versions in the {DTProductName} 3.1
 
 [options="header"]
 |===
 |Operator |Component |Version
-|{JaegerName}
-|Jaeger
-|1.51.0
-
-|xref:../../otel/otel-release-notes.adoc[{OTELName}]
-|OpenTelemetry
-|0.89.0
 
 |{TempoName}
 |Tempo
-|2.3.0
-|===
+|2.3.1
 
-// Jaeger section
-[id="jaeger-release-notes_distributed-tracing-rn-3-0"]
-== {JaegerName}
+//for each new release, update the release number in the xref path on the following line
+|xref:../../otel/otel_rn/otel-rn-3.1.adoc[{OTELName}]
+|OpenTelemetry
+|0.93.0
 
-[id="deprecated-functionality_jaeger-release-notes_distributed-tracing-rn-3-0"]
-=== Deprecated functionality
+|{JaegerName} (deprecated)
+|Jaeger
+|1.53.0
 
-In Red Hat OpenShift distributed tracing 3.0, Jaeger and Elasticsearch are deprecated, and both are planned to be removed in a future release. Red Hat will provide critical and above CVE bug fixes and support for these components during the current release lifecycle, but these components will no longer receive feature enhancements.
+|===
 
-In Red Hat OpenShift distributed tracing 3.0, Tempo provided by the {TempoOperator} and the OpenTelemetry collector provided by the Red Hat build of OpenTelemetry are the preferred Operators for distributed tracing collection and storage. The OpenTelemetry and Tempo distributed tracing stack is to be adopted by all users because this will be the stack that will be enhanced going forward.
+// Tempo section
+[id="distributed-tracing-rn_3-1_tempo-release-notes"]
+== {TempoName}
 
-[id="new-features-and-enhancements_jaeger-release-notes_distributed-tracing-rn-3-0"]
+[id="distributed-tracing-rn_3-1_tempo-release-notes_new-features-and-enhancements"]
 === New features and enhancements
 
-This update introduces the following enhancements for the {JaegerShortName}:
+This update introduces the following enhancements for the {TempoShortName}:
 
-* Support for the ARM architecture.
+* Monolithic custom resource in dev-preview support.
 * Support for cluster-wide proxy environments.
+* Support for TraceQL to Gateway component.
 
-[id="bug-fixes_jaeger-release-notes_distributed-tracing-rn-3-0"]
+[id="distributed-tracing-rn_3-1_tempo-release-notes_bug-fixes"]
 === Bug fixes
 
-This update introduces the following bug fixes for the {JaegerShortName}:
+This update introduces the following bug fixes for the {TempoShortName}:
 
-* Fixed support for disconnected environments when using the `oc adm catalog mirror` CLI command. (link:https://issues.redhat.com/browse/TRACING-3546[TRACING-3546])
+* Fixed operator RBAC for the monitor tab when the operator is deployed in arbitrary namespace. (link:https://issues.redhat.com/browse/TRACING-3786[TRACING-3786])
```

Comment:
Rephrase to follow guidance here: https://redhat-documentation.github.io/supplementary-style-guide/#_bug_fix

For example, maybe something like:

"Before this update, the RBAC configuration for the operator RBAC caused it to deploy in an arbitrary namespace, so RBAC was not explicitly provided to the operator. Now, the operator RBAC for the monitor tab is deployed dynamically based on the namespace of the operator."

I don't know how accurate that is; trying to piece together information from the linked Jira bug, but I don't have the Tempo domain knowledge that you do, so it's just an example.

Reply:
@gwynnemonahan, WDYT?
 
* Before this update, when a TempoStack was created with the `monitorTab` enabled in {product-title} 4.15, the required `tempo-redmetrics-cluster-monitoring-view` ClusterRoleBinding was not created. This update resolves the issue by fixing the operator RBAC for the monitor tab when the operator is deployed in an arbitrary namespace. (link:https://issues.redhat.com/browse/TRACING-3786[TRACING-3786])

* Before this update, the `jaeger-agent` container connection target URL in the `jager-query` pod was overwritten with another namespace URL in {product-title} 4.13. This was caused by a bug in the sidecar injection code in the `jaeger-operator`, causing nondeterministic `jaeger-agent` injection. With this update, the operator prioritizes the Jaeger instance from the same namespace as the target deployment. (link:https://issues.redhat.com/browse/TRACING-3722[TRACING-3722])

* Before this update, the Jaeger Operator used other images than `relatedImages`. This caused the *ImagePullBackOff* error in disconnected network environments when launching the `jaeger` pod because the `oc adm catalog mirror` command mirrors images specified in `relatedImages`. This update provides support for disconnected environments when using the `oc adm catalog mirror` CLI command. (link:https://issues.redhat.com/browse/TRACING-3546[TRACING-3546])

Reply:
@gwynnemonahan, done.